htdocs/core/lib/functions.lib.php

```diff
@@ -38,6 +38,59 @@
 include_once DOL_DOCUMENT_ROOT .'/core/lib/json.lib.php';
 
 /**
+ * Function to transform field from a dictionnary
+ * 
+ * @param 	$db	Database 	Object
+ * @param	$string			String to transform
+ * @param	$tableName		Table of record
+ * @param	$fieldName		Field name
+ */
+function dol_transform_field(&$db, &$string, $tableName, $fieldName) {
```

Comment:
Your documentation is missing the type of each fields which should go _before_ the variable name.
eg:
```php
/**
 * This function does something
 *
 * @param DoliDB $db Database handler
 */
```